test/functional/test_framework/wallet.py

```diff
@@ -55,6 +55,15 @@ def get_utxo(self, *, txid=''):
             index = self._utxos.index(utxo)
         return self._utxos.pop(index)
 
+    def prepare_tx(self, utxo_to_spend, send_value):
```

Comment:
Could we get a more informative function name and/or description? Also I think you should rename to `utxos` and make it a list. "utxo_to_spend" is kinda like "uneaten food to be eaten."

Reply:
😅 my bad. Have never too good at naming functions. But will definitely keep in mind